QKSMS/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
     defaultConfig {
         versionCode 127
         versionName "2.5.5"
-        minSdkVersion 15
+        minSdkVersion 19
```

Comment:
Is this really necessary? This means disabling QKSMS updates for some 22% percent of [market share](https://developer.android.com/about/dashboards/index.html) (4.0.3 - 4.3).